app/src/main/java/org/mozilla/fenix/library/history/HistoryDataSource.kt

```diff
@@ -34,10 +34,6 @@ class HistoryDataSource(
     }
 
     override fun loadBefore(params: LoadParams<Int>, callback: LoadCallback<History>) { /* noop */ }
-
-    companion object {
-        private const val INITIAL_OFFSET = 0
```

Comment:
The same question about moving offset into `ViewModel`. Having a `HistoryDataSource` accessing something from `ViewModel` doesn't look like a very clean data flow solution.

Reply:
Sounds good I'll move this back.